package.json

```diff
@@ -45,14 +45,18 @@
     "perf-sym": "^2.0.3",
     "pump": "^3.0.0",
     "pumpify": "^1.4.0",
+    "read-last-lines": "^1.6.0",
     "semver": "^5.5.1",
     "single-line-log": "^1.0.1",
+    "source-map": "^0.7.3",
     "split2": "^3.0.0",
     "tachyons": "^4.9.1",
     "through2": "^2.0.1",
     "which": "^1.2.4"
   },
   "devDependencies": {
+    "rollup": "^0.66.2",
```

Comment:
can you please pull the rollup dev deps into a separate `package.json` in the `examples/source-maps` folder